### PR TITLE
Feature/resource stock

### PR DESCRIPTION
**Pull request by BenMaruchu**
This PR does the following:
- Add implementation of checkbox both on bulk selection and single selection of stock
- Refactor stock module to have consistency with other modules
- Add export action but for now it's doesn't work since Lally have to fix some issues in the API

### CODE REVIEW

File: src/Stakeholders/components/FocalPeople/List/index.js [hunk 1]
```diff
@@ -211,7 +211,7 @@ class FocalPersonsList extends Component {
               },
               () => {
                 notifyError(
-                  'An Error occurred while refreshing Focal People please focalPerson system administrator'
+                  'An Error occurred while refreshing Focal People please focalPerson co system administrator'
```
Comment (navish) [inline src/Stakeholders/components/FocalPeople/List/index.js]: Check the grammar here.

Comment (BenMaruchu) [reply src/Stakeholders/components/FocalPeople/List/index.js]: This have being fixed in the previos PR for focal people

Comment (navish) [reply src/Stakeholders/components/FocalPeople/List/index.js]: Cool.

**Review by navish — APPROVED** [review]
Looks good.

**Pull request merged**